test/avr/convert.py

```diff
@@ -0,0 +1,183 @@
+import re
```

Comment:
pxt is fundamentally a JS project. We should avoid adding dependencies to Python. Can this be rewritten in JS instead?